### PR TITLE
Incorporate Nav Updates for the introduction of the TFC subpages

### PR DESCRIPTION
**Pull request by BrandonRomano**
CleanShot 2020-12-08 at 09 52 53@2x

## Labels

- [ ] inaccuracy
- [ ] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [x] enhancement - changing the website's behavior/layout

### CODE REVIEW

File: content/source/layouts/_sidebar.erb [hunk 1]
```diff
@@ -6,25 +6,22 @@
...
-    <li><a data-track='intro' href="/intro/index.html">Intro</a></li>
-    <li><a data-track='learn' href="https://learn.hashicorp.com/terraform/">Learn</a></li>
+    <li><a data-track='overview' href="/">Overview</a></li>
+    <li><a data-track='registry' href="https://registry.terraform.io">Registry</a></li>
+    <li><a data-track='learn' href="https://learn.hashicorp.com/terraform/">Tutorials</a></li>
     <li><a data-track='community' href="/community.html">Community</a></li>
-    <li><a data-track='enterprise' href="https://www.hashicorp.com/products/terraform/?utm_source=oss&utm_medium=header-nav&utm_campaign=terraform">Enterprise</a></li>
+    <li><a data-track='github' href="https://github.com/hashicorp/terraform">GitHub</a></li>
+    <li><a data-track='download-cli' href="/downloads.html">Download CLI</a></li>
+    <li><a data-track='sign-up' href="https://app.terraform.io/signup/account?utm_source=docs_banner">Terraform Cloud</a></li>
```
Comment (mwickett) [inline content/source/layouts/_sidebar.erb:15]: I believe this should be linking to `/cloud` (my suggestion here includes changing the `data-track` attribute...I chose `cloud`, but that may not be right)

```suggestion
    <li><a data-track='cloud' href="/cloud">Terraform Cloud</a></li>
```

**Review by mwickett — COMMENTED** [review]
I think the Terraform Cloud button link needs to change, but otherwise it looks good.

**Review by mwickett — COMMENTED** [review]
I think my suggestion (when committed) will remove the extra Terraform Cloud item, as discussed.

**Review by mwickett — APPROVED** [review]

File: content/source/layouts/layout.erb [hunk 2]
```diff
@@ -265,47 +265,47 @@
...
-                      <li><a data-track='learn' href="https://learn.hashicorp.com/terraform/">Tutorials</a></li>
-                      <li><a data-track='community' href="/community.html">Community</a></li>
-                      <li><a data-track='enterprise' href="https://www.hashicorp.com/products/terraform/?utm_source=oss&utm_medium=header-nav&utm_campaign=terraform">Enterprise</a></li>
+                      <%= editions_items %>
                     </ul>
                   </li>
-                  <li class="hidden-sm"><a data-track='intro' href="/intro/index.html">Intro</a></li>
-                  <li class="hidden-sm"><a data-track='learn' href="https://learn.hashicorp.com/terraform/">Tutorials</a></li>
+                  <li class="hidden-sm"><a data-track='registry' href="https://registry.terraform.io">Registry</a></li>
+                  <li class="hidden-sm"><a data-track='learn' href="https://learn.hashicorp.com/terraform/?utm_source=docs_banner">Tutorials</a></li>
```
Comment (BrandonRomano) [inline content/source/layouts/layout.erb:286]:
```suggestion
                  <li class="hidden-sm"><a data-track='learn' href="https://learn.hashicorp.com/terraform/?utm_source=terraform_io">Tutorials</a></li>
```

**Review by nfagerlund — APPROVED** [review]
This looks good to merge for now! There are a few functional glitches I want to see addressed pretty soon post-launch: 

- Right now the /intro section is inaccessible. Mike's looking into whether that's by design, but my understanding prior to this was that we'd just move it into the "docs" menu to get it off the top-level. If possible, I'd like that fixed before merge, but I understand if time is tight. 
- Previously, when the page width shrank to tablet-ish, we'd collapse some of the extra top-level links into an "info" menu, which wasn't present at wider sizes. With the removal of that tablet-size-only overflow menu, the **registry, learn,** and **community** sections are totally unreachable at tablet width (until you go further down to phone-width, at which point they reappear in the burger menu sidebar). We should probably either bring that overflow menu back, or switch to burger-mode earlier. 

Otherwise, this seems good! Excited that we're tightening up the message in the upper zone here.

**Pull request merged**